includes/admin/class-evf-admin-import-export.php

```diff
@@ -62,7 +62,7 @@ public function export_json() {
 		// Force download.
 		header( 'Content-Type: application/force-download' );
 		// Disposition / Encoding on response body.
-		header( "Content-Disposition: attachment;filename={$file_name}" );
+		header( "Content-Disposition: attachment;filename={$file_name};charset=utf-8" );
```

Comment:
Spacing for the charset=utf-8 would be ideal as per w3's RFC standards.

https://www.w3.org/International/articles/http-charset/index

Tests were performed with standard UTF-8 german characters, as well as an emoji.

![image](https://user-images.githubusercontent.com/30156167/75665315-167a7880-5c9c-11ea-914a-0e780ddf2b55.png)

Reply:
Resolved and tested with 71229c5